.github/actions/check/action.yml

```diff
@@ -12,9 +12,22 @@ runs:
 
     - uses: arduino/setup-protoc@v1
       with:
-        version: '3.x'
+        version: "3.x"
         repo-token: ${{ inputs.github_token }}
 
+    - name: Ubuntu install mold and clang
+      shell: bash
+      if: ${{ matrix.config.os == 'ubuntu-latest' }}
+      run: |
+        apt update
+        sudo apt install mold clang libc6
+        echo $(whereis mold)
```

Comment:
I'm exhausted, how to make it run on `ubuntu-latest`?

Reply:
`check` only runs on `ubuntu-latest`.

![image](https://user-images.githubusercontent.com/5351546/154533772-eaab226d-4ce2-472c-8436-3937cd2a8dca.png)

So there is no `matrix.config.os` here, we can use `apt install` directly